precompiles/parachain-staking/src/tests.rs

```diff
@@ -1175,7 +1175,7 @@ fn candidate_request_is_pending_works() {
 				Some(Ok(PrecompileOutput {
 					exit_status: ExitSucceed::Returned,
 					output: Default::default(),
-					cost: 176422000,
```

Comment:
@4meta5 I updated the tests with the new weight value. Could you confirm if this is fine? It feels very wrong to update a magic number via a failing test. Are these weights necessary to be tested here or perhaps we should do a partial equality check without the `cost` field.

Reply:
> I updated the tests with the new weight value. Could you confirm if this is fine?

They all look very close to me (not much difference between the old vs new).

> It feels very wrong to update a magic number via a failing test.

Now it does show us how the cost of these changed. If the change was significant, we would see it here and could investigate further instead of accidentally including a mistake.

Including this field is also how it is done in the other precompile tests (i.e. assets-erc20).

> perhaps we should do a partial equality check without the cost field.

If you can do that in a succinct way, it'd be nice but probably better to propose a change to all precompile tests in a follow up.